integreat_cms/static/src/css/style.scss

```
@@ -561,7 +561,61 @@ datalist option {
 }
 
 /* Special word break for feedback list */
-.feedback-comment-full {
-    @apply break-words whitespace-pre-line;
-    word-break: break-word;
+.feedback-entry {
+    display: fleX;
+    flex-wrap: nowrap;
+    align-items: center;
+
+    span {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: no-wrap;
+    }
+
+    .clickable-icon {
+        display: inline-flex;
+        cursor: pointer;
+        :hover {
+            transform: scale(1.25);
+        }
```

```suggestion
    .toggle-feedback-entry {
        @apply hover:scale-125;
```
If you wrap the icons in `<a>` tags, you also have to define the hover effect on the a elements, otherwise hovering feels a bit choppy and doesn't work reliable. Also, I wouldn't re-create the tailwind classes in plain css...

---

didn't really know that tailwind was behind the css magic happening here. Thx for the note :D